src/MemoryTable.js

```diff
@@ -7,28 +8,67 @@ import ContextureMobx from './utils/contexture-mobx'
 import { componentForType } from './utils/schema'
 import { ResultTable, TypeMap } from './exampleTypes'
 
-export let memoryService = (records, { schema, debug } = {}) =>
-  Contexture({
-    debug,
-    // Hack to effectively set a default schema: if our tree root doesn't have
-    // a `schema` property, it will get the schema at key `undefined`.
-    schemas: { [schema]: { memory: { records } } },
-    providers: { memory: { ...memory, types: types() } },
-  })
-
-let MemoryTable = ({ data, fields, debug, include, ...props }) => {
-  let service = memoryService(data, { schema: 'data', debug })
+export let useMemoryTree = ({
+  records,
+  schema = 'data',
+  fields,
+  debug,
+  resultNode = {
+    pageSize: 50,
+  },
+  criteriaNodes = [],
+  childrenNodes = [],
+} = {}) => {
+  let include = _.map('field', fields)
```

Comment:
I don't think we need the `include` here if the default is to include everything.